composer.json

```diff
@@ -28,24 +28,23 @@
         "forum": "https://octobercms.com/forum/",
         "docs": "https://octobercms.com/docs/",
         "irc": "irc://irc.freenode.net/october",
+        "chat": "https://discord.gg/gEKgwSZ",
         "source": "https://github.com/octobercms/october"
     },
     "require": {
-        "php": ">=7.0.8",
+        "php": "^7.2",
         "ext-mbstring": "*",
         "ext-openssl": "*",
-        "october/rain": "~1.0",
-        "october/system": "~1.0",
-        "october/backend": "~1.0",
-        "october/cms": "~1.0",
-        "laravel/framework": "^5.5.40",
+        "october/rain": "dev-wip/laravel-6",
+        "laravel/framework": "^6.0",
         "wikimedia/composer-merge-plugin": "dev-master"
     },
     "require-dev": {
-        "fzaninotto/faker": "~1.7",
-        "phpunit/phpunit": "~6.5",
-        "phpunit/phpunit-selenium": "~1.2",
-        "meyfa/phpunit-assert-gd": "1.1.0",
+        "fzaninotto/faker": "^1.9",
+        "phpunit/phpunit": "^8.0|^9.0",
+        "phpunit/phpunit-selenium": "dev-master",
```

Comment:
Is this temporary?

Reply:
@LukeTowers I believe so - they seem to be on the cusp of releasing v8.0 which supports PHPUnit 8 and 9. TBH though, I'm not sure if our Selenium tests still work.

Reply:
Perhaps you can add the selenium tests back into the unit testing automation?

Reply:
@LukeTowers I can give it a try - I recall these sort of tests requiring us to fire up a headless version of Chrome to run the tests on, which was painful to install. Were these tests ever part of the automated testing?

Reply:
Not sure, but if we can make it work with github actions that would be pretty sweet. I know there's a couple of plugins that make integrating with Selenium / Dusk in October a bit better, perhaps we should just try and get the current tests working standalone (and publish instructions for doing so) and then later on we can revisit automating them.

Reply:
No worries, I'll add it as a task to the list. I'll leave this thread open as well to remind me.